ubuntu-docker/1204/README.md

```diff
@@ -0,0 +1,24 @@
+# About
+
+This directory contains code to build a docker nmag container based on Ubuntu 12.04.
+
+# Dependencies
+
+* [Docker](https://docs.docker.com/install/)
+
+# Building
+
+In order to build the container run `docker build -t nmag .` from the current directory
+
+# Using
+
+You can execute a command from the container using `docker run nmag <command>`.
+For example:
+```
+docker run nmag nsim
+```
+To execute a command on a file in the current directory or a child of the current directory, use the format `docker run -v $PWD:. nmag <command> <dilename>`.
+For example:
+```
+docker run -v $PWD:. nmag nsim simulation.py
+```
```

Comment:
This command doesn't work for me.

<pre>
base) > $ docker run -v `pwd`:. nmag nsim simulation.py                                                                [±master ✓]
docker: Error response from daemon: invalid volume specification: '/Users/fangohr/git/nmag-containers/ubuntu-docker/1204:.': invalid mount config for type "bind": invalid mount path: '.' mount path must be absolute.
See 'docker run --help'.
</pre>

Should we have a dedicated directory (`/io`) inside the container, to which we mount the local path?

Reply:
Resolved by 7b0d2a452cd64152370ad135287e1188e6b51f1c.